### PR TITLE
fix: Delete message from gallery (AR-1748)

### PR DESCRIPTION
**Pull request by borichellow**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1748" title="AR-1748" target="_blank">BugAR-1748</a>  [Images] Refactor Delete Message Functionality from gallery 
  </td></table>
  <br />
 

### Issues

when use is trying to remove media message from the Gallery screen, app redirects him to the Conversation screen and shows the DeleteMessageDialog instead of showing that dialog in Gallery screen.

Also, all the tests in `MediaGalleryViewModelTest` were not running, because of wrong `@Test` import, it should be `org.junit.jupiter.api.Test`

### Causes (Optional)

it was implemented like this from the beginning

### Solutions

fix it

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-905/1/) **failed**.

File: app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt [hunk 1]
```diff
@@ -311,13 +311,15 @@ enum class NavigationItem(
 
     Gallery(
         primaryRoute = MEDIA_GALLERY,
-        canonicalRoute = "$MEDIA_GALLERY/{$EXTRA_IMAGE_DATA}",
+        canonicalRoute = "$MEDIA_GALLERY/{$EXTRA_CONVERSATION_ID}/{$EXTRA_IMAGE_DATA}",
```
Comment (borichellow) [inline app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt:314]: remove this, it's not needed anymore

File: app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt [hunk 2]
```diff
@@ -311,13 +311,15 @@ enum class NavigationItem(
...
         content = { MediaGalleryScreen() }
     ) {
         override fun getRouteWithArgs(arguments: List<Any>): String {
             val imageAssetId: ImageAsset.PrivateAsset? = arguments.filterIsInstance<ImageAsset.PrivateAsset>().firstOrNull()
             val mappedArgs = imageAssetId?.toString() ?: ""
-            return imageAssetId?.run { "$primaryRoute/$mappedArgs" } ?: primaryRoute
+            val conversationIdString: String = arguments.filterIsInstance<ConversationId>().firstOrNull()?.toString()
```
Comment (borichellow) [inline app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt:320]: and this

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-905/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4287353)(4287353)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4287353)(4287353)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-905/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4297991)(4297991)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4297991)(4297991)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-905/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4298341)(4298341)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4298341)(4298341)-dev-release.apk)

**Review by gongracr — APPROVED** [review]
Awesome refactoring effort 👷 🖼️ 🍫

**Pull request merged**